# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/maim/hotfix-cxx17.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

ICU 7x headers require C++17; maim pins its target to C++11
Signed-off-by: Dennie Mahne <dennie.mahne@gmail.com>

--- maim/CMakeLists.txt	2026-07-19 13:49:21.678158417 +0200
+++ maim/CMakeLists.txt	2026-07-19 13:49:21.682290564 +0200
@@ -38,7 +38,7 @@
 find_package( GLM        REQUIRED )
 
 set_property(TARGET ${BIN_TARGET} PROPERTY CXX_STANDARD_REQUIRED ON)
-set_property(TARGET ${BIN_TARGET} PROPERTY CXX_STANDARD 11)
+set_property(TARGET ${BIN_TARGET} PROPERTY CXX_STANDARD 17)
 set_property(TARGET ${BIN_TARGET} PROPERTY CXX_EXTENSIONS OFF)
 
 # Includes
